src/plugins/intel_gna/transformations/remove_converts.cpp

```diff
@@ -0,0 +1,75 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <openvino/cc/ngraph/itt.hpp>
+
+#include "transformations/remove_converts.hpp"
+
+#include <memory>
+#include <vector>
+
+#include <ngraph/opsets/opset4.hpp>
+#include <ngraph/opsets/opset8.hpp>
+#include <ngraph/pattern/op/wrap_type.hpp>
+#include <ngraph/pass/manager.hpp>
+
+using NodeInput = ngraph::Input<ngraph::Node>;
+using NodeOutput = ngraph::Output<ngraph::Node>;
+
+namespace GNAPluginNS {
+    NGRAPH_RTTI_DEFINITION(RemoveInputConvert, "RemoveInputConvert", 0);
+
+    RemoveInputConvert::RemoveInputConvert() {
+        MATCHER_SCOPE(RemoveInputConvert);
+
+        const auto input = ngraph::pattern::wrap_type<ngraph::opset8::Parameter>();
+        const auto convert = ngraph::pattern::wrap_type<ngraph::opset8::Convert>({input});
```

Comment:
Where check of supported precisions?

Reply:
We are checking them later in the LoadNetwork method (ArePrecisionsSupported), I used the old logic and get the outputs from ngraph function directly

Reply:
added tp transformations